src/app/components/crud/crud.component.ts

```diff
@@ -0,0 +1,132 @@
+import { Component, Input, OnInit } from '@angular/core';
```

Comment:
I would like to suggest a new approach. I would create a new interface here:

`export interface Crud {
  create: (any) => Promise<any>;
  update: (any) => Promise<any>;
  remove: (any) => Promise<any>;
}`

Then I would change the @Input() (create, update, remove) and would create a @Input()crud:Crud.

Finally in teamsAdmin.page.ts and driversAdmin.page.ts would change the methods:

`crud: Crud = {
    create: (team) => this.teamsService.create(team),
    update: (team) => this.teamsService.update(team),
    remove: (team) => this.teamsService.remove(team),
  };`

**crud.component.ts**
![captura de pantalla 2018-02-28 a las 18 26 41](https://user-images.githubusercontent.com/5720287/36802381-ff68bc48-1cb4-11e8-953b-297cd3b90b2c.png)

**teamsAdmin.page.ts**
![captura de pantalla 2018-02-28 a las 18 28 33](https://user-images.githubusercontent.com/5720287/36802476-42c308c2-1cb5-11e8-9f61-4c6294649d4c.png)

**teamsAdmin.page.html**
![captura de pantalla 2018-02-28 a las 18 29 56](https://user-images.githubusercontent.com/5720287/36802523-6b2a8cae-1cb5-11e8-991c-9f35ed72c964.png)


The same with driversAdmin.page